rancher2/resource_rancher2_node_template_test.go

```diff
@@ -267,6 +267,7 @@ resource "` + testAccRancher2NodeTemplateType + `" "foo-vsphere" {
 	cpu_count = "8"
 	disk_size = "20480"
 	pool =  "pool-YYYYYYYY"
+	graceful_shutdown_timeout = "30"
```

Comment:
Nit: set the shutdown timeout as a `const` and use that variable here and on line 746

Reply:
I think I will not move it to const to match other values, like the disk_size, cpu_count.